test/index.test.js

```diff
@@ -1117,4 +1117,225 @@ describe('index', () => {
             });
         });
     });
+
+    describe('_addWebhook', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'bitbucket.org:arnoldWeber/wyattLib:mybranch';
```

Comment:
Can we change to `bitbucket.org:arnoldWeber/{1234}:mybranch` to make it more real? (scm-bitbucket has this format for scmUri)